CHANGELOG.md

```diff
@@ -4,12 +4,84 @@
+  See https://docs.voltocms.com/upgrade-guide/ for more information.
+- Lazyload draftjs library. See the upgrade guide on how that impacts you, in case you have extended the rich text editor configuration @tiberiuichim @kreafox
+  See https://docs.voltocms.com/upgrade-guide/ for more information.
+- Deprecating `lang` cookie in favor of Plone official one `I18N_LANGUAGE` @sneridagh
+
 ### Feature
 
+- Add `cookiesExpire` value to config to control the cookie expiration @giuliaghisini
+- DatetimeWidget 'noPastDates' option: Take widgetOptions?.pattern_options?.noPastDates of backend schema into account. @ksuess
+- Add a new type of filter facet for the Search block. Heavily refactor some searchblock internals. @tiberiuichim
+- Add date range facet to the search block @robgietema
+- Introduce the new `BUILD_DIR` runtime environment variable to direct the build to run in a specific location, different than `build` folder. @sneridagh
+- Handle redirect permanent calls from the backend in the frontend (e.g. when changing the short name) @robgietema
+- Added id widget to manage short name @robgietema
+- Refactor language synchronizer. Remove it from the React tree, integrate it into the Api Redux middleware @sneridagh
+- Add blocks rendering in Event and NewsItem views (rel plone.volto#32) @nzambello @ksuess
+- Add internal volto ids to invalid ids @robgietema
+- Complete basque translation @erral
+- Complete spanish translation @erral
+- Sort the choices in Facets in the search block @iFlameing
+
 ### Bugfix
 
+- Fix the `null` error in SelectAutoComplete Widget @iFlameing
+- Prevent the MultilingualRedirector to force 4 content load when switching the language @reebalazs
+- Fix the upload image in contents view @iFlameing
+- add "view" id to contact-form container for main content skiplink @ThomasKindermann
+- Fix loading indicator positioning on Login form submit @sneridagh
+- Fix redirect bug with URLs containing querystrings @robgietema
+- Fixed id widget translations @robgietema
+- Contents Rename Modal, use `id` Widget type @sneridagh
+- Fix overflow of very long file name in `FileWidget` @sneridagh
+- Fix overflowing issue in the toolbar @kreafox
+
 ### Internal
 
+- Change prop `name` -> `componentName` in component `Component` @sneridagh
+- Add new RawMaterial Volto websites in production @nzambello
+- House cleanup, remove some unused files in the root @sneridagh
+- Move Webpack related files to `webpack-plugins` folder @sneridagh
+- Remove unused Dockerfiles @sneridagh
+- Update Docker compose to latest images and best practices @sneridagh
+- Improve flaky test in coresandbox search Cypress tests @sneridagh
+- Better implementation of the add-on load coming from the environment variable `ADDONS` @sneridagh
+- Turn `lazyLibraries` action into a thunk. Added a conditional if the library is loaded or in process to be loaded, do not try to load it again. This fixes the lag on load `draftjs` when having a lot of draftjs blocks. @sneridagh
```

Comment:
"thunk"? perhaps "chunk"?

Reply:
it's fine, this artifact is called like this.